sympy/physics/units/unitsystem.py

```diff
@@ -132,9 +132,6 @@ def dim(self):
         """
         return len(self._base_units)
 
-    def get_dimension_system(self):
-        return self._dimension_system
-
```

Comment:
Isn't this a public method? Why delete?

Reply:
It's duplicated above https://github.com/sympy/sympy/pull/17869/files#diff-b0b3e4d2c5be1d62ff3d6397a2556ad2R92